ApollosProject/apollos-prototype
# Adds Script to Generate Component Docs

## DESCRIPTION

![Kapture 2019-03-20 at 13 12 20](https://user-images.githubusercontent.com/2659478/54704992-3871fe00-4b12-11e9-822e-d2957d3f17fc.gif)

### What does this PR do, or why is it needed?

This will take JSDoc descriptions from our components and spit out markdown files with prop definitions that we can use on our doc site. It also includes a step in Travis that will automatically merge new docs into the site. Fully automated 🙌 

### What design trade-offs/decisions were made?

### How do I test this PR?

Add a JSDoc to one of our ui-kit components and then run `yarn generate-docs` from the root. Then check `packages/apollos-ui-kit/docs/generated/` for your new doc.

### What automated tests did you write?

## TODO:

- [x] Closes #592

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
